pkg/stanza/docs/operators/README.md

```diff
@@ -0,0 +1,46 @@
+## What is an operator?
+An operator is the most basic unit of log processing. Each operator fulfills a single responsibility, such as reading lines from a file, or parsing JSON from a field. Operators are then chained together in a pipeline to achieve a desired result.
+
+For instance, a user may read lines from a file using the `file_input` operator. From there, the results of this operation may be sent to a `regex_parser` operator that creates fields based on a regex pattern. And then finally, these results may be sent to a `elastic_output` operator that writes each line to Elasticsearch.
```

Comment:
```suggestion
For instance, a user may read lines from a file using the `file_input` operator. From there, the results of this operation may be sent to a `regex_parser` operator that creates fields based on a regex pattern. And then finally, these results may be sent to a `stdout` operator that writes each line to stdout.
```

This is probably minor, but elastic is no longer included among the outputs. Also, I'm wondering if we should describe that the library is used in the receiver which means it does not need to use outputs normally

Reply:
Good feedback, this doc is quite out of date. I took a pass through the document and updated the contents to reflect the current state much more accurately.